### ISSUE TITLE
TS Server not picking up new TS files in vscode on MacOS

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#33564]
**TypeScript Version:**  3.7.0-dev.20190922

**Search Terms:**
- create-react-app
- project configuration
- tsconfig

**Repo**
1. `npx create-react-app  my-app --typescript`
1. Open `my-app` in VS Code
1. Open `app.tsx`
1. Now create a new file next to it called `add.ts` with the content:

    ```ts
    export function add() {}
    ```

1. Run the `go to project` configuration command in VS Code with `add.ts` open:
     * 🐛 Says it's not part of any configured project

1. Now go back to `App.tsx` and try using completions to add an auto import for `add`
     * 🐛 Auto import never works (unless you add an import manually)

Here are the logs:
[log.txt](https://github.com/microsoft/TypeScript/files/3643560/log.txt)

**Comment by mjbvz** [comment]
This repos for me with 3.6.3 and for very simple projects like the following:

`tsconfig.js`

```
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es2016",
        "jsx": "react"
    }
}
```

`src/main.ts`
```ts
export function foo() {
    add
}
```

`src/add.ts` (must be newly created)
```ts
export function add() { }
```

I don't know why we haven't seen this reported on the VS Code side (to my knowledge) but this bug is really annoying. @RyanCavanaugh @sheetalkamat I think this is worth at least investigating for a 3.6.x recovery

**Comment by sheetalkamat** [comment]
@mjbvz in the logs you showed there is only `updateOpen` but directory watcher for wild card directories is never invoked and that's the reason project doesn't know about `app.tsx` being part of it.. 

```
Info 80   [11:20:43.245] request:
    {"seq":14,"type":"request","command":"updateOpen","arguments":{"changedFiles":[],"closedFiles":[],"openFiles":[{"file":"/Users/matb/projects/san/sub/my-app/src/add.ts","fileContent":"","scriptKindName":"TS","projectRootPath":"/Users/matb/projects/san/sub/my-app"}]}}

```

When I repro your scenario (simple one) I don't see that behavior...

![import](https://user-images.githubusercontent.com/8052792/65464300-4df01980-de0e-11e9-9d01-50a41a60de03.gif)

**Comment by mjbvz** [comment]
This was with the latest VS Code 1.39 insiders for me with MacOS 10.14.6. I've been able to repo the create-react-app issues it on two macs. Let me give it a quick try on my windows vm too

**Comment by mjbvz** [comment]
Seems to work ok on windows using create-react-app

**Comment by mjbvz** [comment]
This could be related to our electron update (which goes from node 10 to node 12.4). I can't seem to repo with the current stable VS Code builds using typescript@next

It definitely happens pretty consistently on mac VS Code insiders for me though. @RyanCavanaugh need to investigate this before VS Code 1.39 ships

**Comment by mjbvz** [comment]
We have reverted our Electron update for this release as this bug also effects other extensions

Closing this TS side issue in favor of the VS Code issue: https://github.com/microsoft/vscode/issues/81516

**Comment by danilofuchs** [comment]
This is still an issue for me, running the latest stable VSCode version. @mjbvz was there any regressions since it was fixed in electron 6?

**Comment by mjbvz** [comment]
@danilofuchs Please open a new issue
